thrifty-schema/src/main/java/com/microsoft/thrifty/schema/Field.java

```diff
@@ -54,6 +54,7 @@ private Field(Builder builder) {
         this.fieldNamingPolicy = builder.fieldNamingPolicy;
         this.annotations = builder.annotations;
         this.type = builder.type;
+        this.javaName = builder.javaName;
```

Comment:
I wouldn't recommend adding `javaName` to builders; it's a computed field, determined by the thrift name and the naming policy.

In fact, this field is going to go away soon as part of the work on #69; probably not worth spending much time on it.